packages/core/src/Button/Button.js

```diff
@@ -65,22 +78,71 @@ const variations = {
         props.disabled
           ? ''
           : `
-        color: ${getPaletteColor('dark')(props)};
-        box-shadow: inset 0 0 0 2px ${getPaletteColor('dark')(props)};
-      `}
+      color: ${getPaletteColor('dark')(props)};
+      box-shadow: inset 0 0 0 2px ${getPaletteColor('dark')(props)};
+    `}
     }
   `,
-  link: css`
-    -webkit-font-smoothing: inherit;
-    font-weight: ${(props) => props.theme.fontWeights.medium};
-    color: ${getPaletteColor('base')};
-    line-height: ${(props) => props.theme.lineHeights.standard};
-    vertical-align: inherit;
-    padding: 0;
+  plain: css`
     background-color: transparent;
+    color: ${(props) => getPaletteColor('primary.base')(props)};
```

Comment:
Do we want to limit these new variations to always use the same color, instead of the passed in color? What's the purpose of adding `primary` and `secondary` variations when it's the same as (minus the focus state, but that seems like it could be added to `fill`):
```
<Button /> = <Button variation="primary" />
<Button color="secondary" /> = <Button variation="secondary" />
```
Also, don't these new variations make the `color` prop redundant? `<Button variation="secondary" color="primary" />` etc?